core/store/src/metrics.rs

```diff
@@ -273,6 +273,22 @@ pub static FLAT_STORAGE_DISTANCE_TO_HEAD: Lazy<IntGaugeVec> = Lazy::new(|| {
     )
     .unwrap()
 });
+pub static FLAT_STORAGE_VALUE_REF_CACHE_LEN: Lazy<IntGaugeVec> = Lazy::new(|| {
+    try_create_int_gauge_vec(
+        "flat_storage_value_ref_cache_len",
+        "Number of items in flat storage cache for its head",
+        &["shard_id"],
+    )
+    .unwrap()
+});
+pub static FLAT_STORAGE_VALUE_REF_CACHE_TOTAL_KEY_SIZE: Lazy<IntGaugeVec> = Lazy::new(|| {
+    try_create_int_gauge_vec(
+        "flat_storage_value_ref_cache_total_key_size",
+        "Total size of all keys in flat storage cache for its head",
```

Comment:
Out of curiosity why do you only measure the size of the keys and not keys and values?

Reply:
Because size of values is fixed - we store `ValueRef`s instead of values.

Reply:
Should we measure both since we might implement inlining in the future?